src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/LinkDesignerBase.cs

```diff
@@ -60,8 +69,19 @@ protected bool FindResourceDesigner (AssemblyDefinition assembly, bool mainAppli
 
 				}
 			}
-			if (string.IsNullOrEmpty(designerFullName))
+			if (string.IsNullOrEmpty(designerFullName)) {
+				// Check for known designers which have been removed.
+				LogMessage ($"    c");
+				foreach (var d in knownDesigners) {
+					LogMessage ($"    Checking {d} against {assembly.Name.Name}");
+					if (d.Contains (assembly.Name.Name)) {
```

Comment:
`"Microsoft.Maui".Contains(assembly.Name.Name)` looks "weird" to me; `"Microsoft.Maui".Contains("M")` is True, so if anyone passes in an assembly named `M`, it'll pass.

Do we really want `.Contains()`, and not `==`?

If we *do* want `.Contains()`, then we *don't* need a `Microsoft.Maui` entry, because that's a substring of `Microsoft.Maui.Core` and `Microsoft.Maui.Controls`…